sql/core/src/test/scala/org/apache/spark/sql/streaming/test/DataStreamReaderWriterSuite.scala

```diff
@@ -21,6 +21,7 @@ import java.io.File
 import java.util.Locale
 import java.util.concurrent.TimeUnit
 
+import scala.compat.Platform.ConcurrentModificationException
```

Comment:
java.util.ConcurrentModificationException?

Reply:
woops... yeah was definitely supposed to be that one :).